lib/cli/run-helpers.js

```diff
@@ -164,8 +164,16 @@ exports.handleFiles = ({
   });
 
   if (!files.length) {
-    console.error(ansi.red(`${symbols.error} No test files found`));
+    // print messages as an error
+    errors.forEach(message => {
+      console.error(ansi.red(`Error: ${message}`));
+    });
```

Comment:
This should have simply said:
```js
    console.error(ansi.red('Error: No test files found'));
```
Nothing is gained by saying more and now we have **both** warnings and errors with the same message.

Reply:
They don't show at the same time though. I mentioned in my comment earlier I felt giving full error details was helpful, in that I found it useful myself.

Reply:
I'm a bit confused about when we are going to show errors vs warnings...

Reply:
Wasn't the name of the issue "too much noise if no files found"?

Reply:
I just didn't want to see a warning about it (which duplicated the filenames) and then *another* error.  If that's fixed, then great

Reply:
Check the description I tried to give full details on what shows when